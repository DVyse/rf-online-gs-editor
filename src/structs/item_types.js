export const FACE = 'face';
export const UPPER = 'upper';
export const LOWER = 'lower';
export const GAUNTLET = 'gauntlet';
export const SHOE = 'shoe';
export const HELMET = 'helmet';
export const WEAPON = 'weapon';
