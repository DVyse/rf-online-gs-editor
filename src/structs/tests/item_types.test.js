import * as itemTypes from '../item_types';

describe('Structs', () => {
  describe('ItemTypes', () => {
    it('should have itemTypes', () => {
      expect(itemTypes).toMatchSnapshot();
    });
  });
});
