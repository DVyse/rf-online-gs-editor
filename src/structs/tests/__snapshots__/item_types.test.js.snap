// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Structs ItemTypes should have itemTypes 1`] = `
Object {
  "AMULET": "amulet",
  "ANIMUS": "animus",
  "BAG": "bag",
  "BATTERY": "battery",
  "BOOTY": "booty",
  "BOX": "box",
  "BULLET": "bullet",
  "CASHMINING": "cashmining",
  "CLOAK": "cloak",
  "COMBINEDATA": "combinedata",
  "COUPON": "coupon",
  "DUNGEONKEY": "dungeonkey",
  "FACE": "face",
  "FIRECRACKER": "firecracker",
  "FORCE": "force",
  "GAUNTLET": "gauntlet",
  "HELMET": "helmet",
  "LOWER": "lower",
  "MAKEDATA": "makedata",
  "MAKETOOL": "maketool",
  "MAP": "map",
  "MAUARM": "mauarm",
  "MAUBACK": "mauback",
  "MAUBULLET": "maubullet",
  "MAUHEAD": "mauhead",
  "MAULOWER": "maulower",
  "MAUSHOULDER": "maushoulder",
  "MAUUPPER": "mauupper",
  "ORE": "ore",
  "PAGER": "pager",
  "POTION": "potion",
  "QUEST": "quest",
  "RADAR": "radar",
  "RECOVERY": "recovery",
  "RESOURCE": "resource",
  "RING": "ring",
  "SHIELD": "shield",
  "SHOE": "shoe",
  "SIEGEKIT": "siegekit",
  "TICKET": "ticket",
  "TOWER": "tower",
  "TOWN": "town",
  "TRAP": "trap",
  "UNITKEY": "unitkey",
  "UNK3": "unk3",
  "UPPER": "upper",
  "WEAPON": "weapon",
}
`;
